### DISCUSSION TITLE
onDrag events not working for the Map.

### DISCUSSION THREAD

**Discussion by AbdelrahmanAhmed605** [giorgiabosello/google-maps-react-markers#257] [Help]
When using the `GoogleMap` component, I cannot get any of the dragging events to work.

```
<GoogleMap
  events={[
          {
            name: 'onDrag',
            handler: () => {
              console.log('a')
            },
          },
          {
            name: 'onDragStart',
            handler: () => {
              console.log('b')
            },
          },
         {
            name: 'onDragEnd',
            handler: () => {
              console.log('c')
            },
          },
  ]}
>
...
</GoogleMap>
```

Any ideas why this is, or any workarounds that may work?

**Comment by giorgiabosello** [comment]
@AbdelrahmanAhmed605 Those events are for the `div`. The drag events need to be passed to the Marker itself. You have the example on the [readme](https://github.com/giorgiabosello/google-maps-react-markers?tab=readme-ov-file#-usage) :)
